monitoring-custom-metrics/deployment/service-monitor.yaml

```diff
@@ -16,5 +16,11 @@ spec:
   endpoints:
     - port: web
```

Comment:
you renamed the port, have you tested the setup?

Reply:
Tested it now, was indeed an error